### PR TITLE
Add option to suppress browser

### PR DESCRIPTION
**Pull request by pkimes**
Added optional parameter to prevent URLs from being opened in browser (default preserves current behavior: `browse = TRUE`). Solves issue #24 by calling `print` with `options(browser = "false")` when user doesn't want URLs to open.

Also includes small change to Urban Dictionary output to remove extra newline between "Tags" and "Link".

### CODE REVIEW

**Comment by jimhester** [comment]
I think maybe a better API would be to add a browse argument to the `print()` method and possibly set the default to a global option.

```r
print <- function(x, browse = getOption("available.browse", TRUE)) { 
 # ...
}
```

That way if people don't like the default behavior they can set it in their `.Rprofile` to never browse or enable on a case by case basis.

**Comment by pkimes** [comment]
Thanks for taking a look at this! :sob: of joy!

Maybe I'm missing something, but wouldn't passing `browse` as on option to `print()` require explicitly calling `print()` at the end of `available()` to get the desired effect, i.e. here:
https://github.com/ropenscilabs/available/blob/e12260e2587d067e11799593bf829a8e458a9368/R/available.R#L30

I'm guessing this would be a bit odd if someone wanted to call `print()` outside of `available()`, e.g.
```{r}
res <- available("xzibit", browse = FALSE)
res ## still opens browser
```
(not really sure why anyone would do this, but you never know...)

As an alternative, would adding `browse` as an attribute (like `packagename`) be acceptable? `print()` would just check `attr(x, "browse")` to decide what to do.

**Comment by jimhester** [comment]
Adding browse as an attribute would be a good solution 👍

**Comment by pkimes** [comment]
Great - thanks! I'll close this pull request and open a separate one to keep the commit history cleaner.

**Comment by jimhester** [comment]
I would actually prefer you keep everything in one PR. I can rebase your commits before I merge if needed to clean up history, or you can do so and `git push --force-with-lease` to this PR.

**Comment by pkimes** [comment]
Sorry - I should have asked.
I rebased and added the change to this branch (but didn't clear out the older commits).

File: R/available.R [hunk 1]
```diff
@@ -27,11 +31,29 @@ available <- function(name, ...) {
...
+  }
+  cat(boxes::rule(attr(x, "packagename")), "\n", sep = "")
+  for (i in x) {
+    print(i)
+  }
+  invisible(x)
+}
+
+#' @export
+print.available_silent_query <- function(x, ...) {
```
Comment (jimhester) [inline R/available.R]: You can get rid of this method now as well.

**Comment by jimhester** [comment]
Thanks! 

I merged this manually at https://github.com/ropenscilabs/available/commit/e39682aaf8d1386155df545e5550124b325f8fa8

**Pull request closed without merging**